pensjon-brevbaker/build.gradle.kts

```diff
@@ -89,6 +89,7 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-html-jvm:0.8.0")
 
     implementation(project(":template-model-generator"))
+    implementation(project(mapOf("path" to ":pensjon-brevbaker-api-model")))
```

Comment:
fjern denne linjen

Reply:
Fjernet "implementation(project(mapOf("path" to ":pensjon-brevbaker-api-model")))"